modules/phase_field/src/kernels/CHSplitConcentration.C

```diff
@@ -0,0 +1,54 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "CHSplitConcentration.h"
+
+template<>
+InputParameters validParams<CHSplitConcentration>()
+{
+  InputParameters params = validParams<Kernel>();
+  params.addClassDescription("Concentration kernel in Split Cahn-Hilliard that solves chemical potential in a weak form");
+  params.addRequiredParam<MaterialPropertyName>("mobility", "Mobility property name");
+  params.addRequiredCoupledVar("chemical_potential_var", "Chemical potential variable");
+
+  return params;
+}
+
+CHSplitConcentration::CHSplitConcentration(const InputParameters & parameters) :
+    DerivativeMaterialInterface< Kernel >(parameters),
+    _mobility_name(getParam<MaterialPropertyName>("mobility")),
+    _mobility(getMaterialProperty<RealTensorValue>(_mobility_name)),
```

Comment:
This just always uses a tensor for the mobility. We have this in a few classes that we name `...Aniso`. I suggest to keep this naming scheme. This will allow us to refactor this into a templated base class that we can use to make a varant with a `Real` mobility as well (which would then have the name `CHSplitConcentration` as opposed to `CHSplitConcentrationAniso`).